docusaurus/docs/learn/quickstarts/browzer/index.md

```diff
@@ -0,0 +1,201 @@
+then you can make an informed decision if you want to modify the default authentication policy or not
+:::
+
+```bash
+ext_jwt_signer="__ext_jwt_signer_id_from_above__"
+ziti edge create auth-policy \
+  "auth-policy-name-here" \
+  --primary-ext-jwt-allowed \
+  --primary-ext-jwt-allowed-signers ${ext_jwt_signer}
+```
+
+### Associate Identities to the Authentication Policy
+
+With an auth policy created and associated to the external jwt signer, you are now ready to create or modify identities
+with this new authentication policy. To allow a given identity to be enabled for BrowZer, you will need to first
+update (or create) the identity with an `externalId` and you will need to associate the identity with the authentication
+policy.
+
+In this example, a new identity will be created and the associated OIDC provider will be expected to provide a bearer
+token which has an `email` claim with the value `ziggy@openziti.io` and names the identity `openziti_ziggy`:
+```bash
+identity_name="openziti_ziggy"
+auth_policy="__auth_policy_id_from_above__"
+external_id="ziggy@openziti.io"
+ziti edge create identity user "${identity_name}" \
+  --auth-policy ${auth_policy} \
+  --external-id "${external_id}" \
+  -a docker.whale.dialers
+```
+
+## Running the Ziti BrowZer Bootstrapper
+
+Running the Ziti BrowZer Bootstrapper can be done directly using NodeJS or with a container engine such as docker. Either
+way you choose to run it, you will need to establish some environment variables. If you are using NodeJS, you'll `export`
+them. For Docker-based running, you can eitehr reference the environment variables (shown below) or use a `.env` file
+if you prefer.
+
+* `NODE_ENV`: controls if the environment is `production` or `development`
+* `ZITI_BROWZER_RUNTIME_LOGLEVEL`: the log level for the Ziti BrowZer Runtime (ZBR) to use. `trace|debug|info|warn|error`
+* `ZITI_BROWZER_RUNTIME_HOTKEY`: the hotkey to activate the BrowZer settings dialog modal. default: alt+F12
+* `ZITI_CONTROLLER_HOST`: the "alternative" address for the OpenZiti controller. example: `ctrl.openziti.io`
+* `ZITI_CONTROLLER_PORT`: the port to find the OpenZiti controller at. example: `8441`
+* `ZITI_AGENT_LOGLEVEL`: the log level for the ziti-http-agent to log at. `trace|debug|info|warn|error`
+* `ZITI_AGENT_HOST`: the address the ziti-http-agent is available at. example: `browzer.openziti.io`
+* `ZITI_AGENT_LISTEN_PORT`: the port the ziti-http-agent is available at. example: `443`
+* `ZITI_AGENT_SCHEME`: the scheme to use to access the ziti-http-agent. `http|https` (https by default)
+* `ZITI_AGENT_CERTIFICATE_PATH`: the path to the certificate the ziti-http-agent presents to clients
+* `ZITI_AGENT_KEY_PATH`: the associated key for the ZITI_AGENT_CERTIFICATE_PATH
+* `ZITI_AGENT_TARGETS`: A json block representing the services to enable BrowZer for. __more on this below__
+
+```bash
+      NODE_ENV: production
```

Comment:
Indented intentionally for a reason, e.g., to opt-out of shell history?

Reply:
nah. likely a copy/paste issue.

Reply:
oh, actually it's probably so it can be copied directly into a compose file with proper indentation